### ISSUE TITLE
a bug with the partial cognate annotation

### ISSUE THREAD

**Issue by nh36** [digling/burmish#55]
Suppose there are two morphemes A B, if I try to associate B with a cognate when A is not yet associated, this causes problems. It can either lead to error messages or two two IDs being associated with a morpheme.

**Comment by LinguList** [comment]
Yep, what needs to be done is: assign as many 0s as there are morphemes in each set that is empty and currently being worked on (all concepts in selection).

**Comment by LinguList** [comment]
Has been redirected to this issue on the edictor: https://github.com/digling/edictor/issues/102 

Once handled there, I'll also close this one.
